### PR TITLE
CNV-9288 RN Preallocation feature & known issue

### PR DESCRIPTION
**Pull request by aburdenthehand**
Added known issue for the preallocation exceptions. 
Also since there was no feature RN for this feature have added that in the Storage New Features section.
@ousleyp FYI

### CODE REVIEW

**Review by aglitke — COMMENTED** [review]
Looks good to me but would appreciate a review from @tomob

File: virt/virt-2-6-release-notes.adoc [hunk 1]
```diff
@@ -82,7 +83,10 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //New known issues for 2.6 (WIP)
 
-//CNV-9288: Blank block disks and VMware imported disk images are not preallocated even if requested
+* Some CDI operations are currently not preallocated when requested. These include:
```
Comment (sjhala-ccs) [inline virt/virt-2-6-release-notes.adoc:86]: Is it possible to link to the related BZ or documentation for this issue?

File: virt/virt-2-6-release-notes.adoc [hunk 2]
```diff
@@ -60,6 +60,7 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-9456: When importing VM disk images, you can now control which multus network is used
 
+* The CDI can now xref:../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[preallocate disk space] to improve write performance of importing and uploading operations and when creating blank data volumes.
```
Comment (sjhala-ccs) [inline virt/virt-2-6-release-notes.adoc:63]: This sentence could be rewritten for clarity. Maybe something like "The Containerized Data Importer (CDI) can now preallocate disk space to import and upload data at a faster speed and when creating blank data volumes."

**Review by sjhala-ccs — COMMENTED** [review]
Couple of suggestions, otherwise LGTM

File: virt/virt-2-6-release-notes.adoc (hunk 1 above)
Comment (aburdenthehand) [reply virt/virt-2-6-release-notes.adoc:86]: No bug to speak of (I think it was issue within epic) but I can add an xref to our preallocation docs

**Review by jeana-redhat — APPROVED** [review]
approving exception

**Pull request merged**